maestro-orchestra/src/test/java/maestro/orchestra/MaestroCommandTest.kt

```diff
@@ -33,7 +33,7 @@ internal class MaestroCommandTest {
     @Test
     fun `description (with a label)`() {
         // given
-        val maestroCommand = MaestroCommand(SetLocationCommand(12.5266, 78.2150, "Set Location to Test Laboratory"))
+        val maestroCommand = MaestroCommand(SetLocationCommand("12.5266", "78.2150", "Set Location to Test Laboratory"))
```

Comment:
Can you write a test using negative coordinates ?

Reply:
@Alaksion done. Please review.

Reply:
thanks!

Reply:
@Alaksion please approve the PR if everything looks good. Thanks.